tests/test_naive_completion.py

```diff
@@ -49,42 +51,46 @@ def test_function_name_completion(completer, complete_event):
 def test_column_name_completion(completer, complete_event):
     text = 'SELECT  FROM users'
     position = len('SELECT ')
-    result = set(completer.get_completions(
+    result = completions_to_set(completer.get_completions(
         Document(text=text, cursor_position=position),
         complete_event))
-    assert result == set(map(Completion, completer.all_completions))
+    assert result == completions_to_set(
+        map(Completion, completer.all_completions))
 
 
+@pytest.mark.skip
```

Comment:
This is also broken in mycli, do you want to keep this test @j-bennet ?

Reply:
No. I should not have put this test in there in the first place. Please kill it.